cli/lidoStatus_test.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2022 Nethermind
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cli
+
+import (
+	"bytes"
+	"io"
+	"math/big"
+	"testing"
+
+	"github.com/shopspring/decimal"
+	log "github.com/sirupsen/logrus"
+)
+
+type flags struct {
+	rewardAddress    string
+	networkName      string
+	longDescriptions bool
+}
+
+type lidoStatusCmdTestCase struct {
+	name  string
+	flags flags
+	isErr bool
+}
+
+func (f flags) argsList() []string {
+	s := make([]string, 0)
+	s = append(s, f.rewardAddress)
+	if f.networkName != "" {
+		s = append(s, "--network", f.networkName)
+	}
+	if f.longDescriptions {
+		s = append(s, "--l")
+	}
+
+	return s
+}
+
+func TestLidoStatusCmd(t *testing.T) {
```

Comment:
Please cover the cases when the user could provide a random string as an argument, an address without the `0x` prefix, or an invalid address such as `0x666ABCDEF`

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/402/commits/7880b17ce9c0c5689a9bf5f6e92a300813f2b95e)